mesheryctl/internal/cli/root/system/model.go

```diff
@@ -413,6 +425,133 @@ mesheryctl exp model search [query-text]
 	},
 }
 
+var importModelCmd = &cobra.Command{
```

Comment:
Should we break it into different files? as it's becoming too big.
```
models/
    import
    list
    ....
```

Reply:
I recommend this one change. Would become easy to track